src/AccessibilityInsights/Modes/LiveModeControl.xaml.cs

```diff
@@ -180,7 +180,7 @@ public async Task SetElement(Guid ecId)
 
                     this.ctrlHierarchy.IsEnabled = false;
                     ctrlHierarchy.Visibility = Visibility.Visible;
-                    spInstructions.Visibility = Visibility.Collapsed;
+                    svInstructions.Visibility = Visibility.Collapsed;
```

Comment:
New since 720: Collapse ScrollViewer, not just StackPanel

Reply:
If you wanted to get fancy, you could introduce a new converter like `InvertVisibilityConverter` in the same style of the existing `NegateBooleanConverter` and bind `svInstructions.Visibility` to the `InvertVisibilityConverter`'ed `ctrlHierarchy.Visibility`. Just an option!